sys-kernel/genkernel/genkernel-9999.ebuild

```diff
@@ -128,11 +128,24 @@ RDEPEND="${PYTHON_DEPS}
 	sys-devel/libtool
 	virtual/pkgconfig
 	elibc_glibc? ( sys-libs/glibc[static-libs(+)] )
-	firmware? ( sys-kernel/linux-firmware )"
-
-if [[ ${PV} == 9999* ]]; then
-	DEPEND="${DEPEND} app-text/asciidoc"
-fi
+	firmware? ( sys-kernel/linux-firmware )
+	plymouth? ( sys-boot/plymouth )
+	doc? ( app-text/asciidoc )"
+	# iscsi ? ( net-libs/libiscsi )
+	# dmraid ? ( sys-fs/ )
```

Comment:
I'd leave these out for now / remove it from the final commit, even fi it's what we're adding towards.

(Also, the syntax is `x? ( ... )`, not `x ? ( ... )`).

But for now, we can make the distfiles conditionally downloaded depending on which features are enabled, right?

Reply:
Thanks! I'll change it accordingly!